package.json

```diff
@@ -15,6 +15,7 @@
   "author": "visualfanatic",
   "main": "index.js",
   "dependencies": {
-    "svgo": "^0.7.1"
+    "svgo": "^0.7.1",
+    "vue-template-compiler": "^2.4.2"
```

Comment:
Oh, possibly yeah. What version of vue/vue-loader are you using?

Is published here for now till this gets merged: https://www.npmjs.com/package/@teamwork/vue-svg-loader